src/views/UserView.vue

```diff
@@ -1,15 +1,15 @@
 <template>
-  <user-home />
+  <user-available-res />
```

Comment:
This page may not just be what is currently on it. It is the user's home with reservations available to them but it may be more than that in the future, so for now just leave it as user-home so it's clear that it is the user's landing component.